java-bootcamp/src/exercise4_builder/OracleConnectionBuilder.java

```diff
@@ -0,0 +1,26 @@
+package exercise4_builder;

```

Comment:
In Java packages have a reversed URL format, something like com.globant.controller, please try to use that kind of packages, we could use comm.globant.bootcamp.exercise1 or something like that